build-tools/android-toolchain/android-toolchain.targets

```diff
@@ -122,19 +122,19 @@
       DependsOnTargets="_SetMxeToolchainMakefileTimeToLastCommitTimestamp"
       Condition="$(AndroidSupportedHostJitAbisForConditionalChecks.Contains (':mxe-Win64:'))"
       Inputs="..\..\external\mxe\Makefile"
-      Outputs="$(AndroidMxeFullPath)\bin\i686-w64-mingw32.static-gcc">
+      Outputs="$(AndroidMxeFullPath)\bin\i686-w64-mingw32.static-gcc;$(AndroidMxeFullPath)\bin\i686-w64-mingw32.static-cmake">
```

Comment:
we added cmake and  zlib to the command 

https://github.com/xamarin/xamarin-android/pull/89/files#diff-84fb09c4b62dc7398bf9a0683da5e15fR127